<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="E7841-Tech" name="UseLoginTest" tests="1" failures="1" timestamp="27 Jan 2020 10:06:28 GMT" time="0.332" errors="0">
  <testcase name="UserLoginAPI_Test" time="0.209" classname="com.LearningAPI.TestCases.UserLoginAPI">
    <failure type="java.lang.AssertionError" message="Validation Response_param is :(),Actual Response_paramvalue({status=0, message=Oops!! checksum error, please check your parameters.})did not match with Expected Response_paramvalue() expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Validation Response_param is :(),Actual Response_paramvalue({status=0, message=Oops!! checksum error, please check your parameters.})did not match with Expected Response_paramvalue() expected [true] but found [false]
	at com.LearningAPI.TestCases.BaseTest.postExecutionvalidation(BaseTest.java:99)
	at com.LearningAPI.TestCases.UserLoginAPI.UserLoginAPI_Test(UserLoginAPI.java:104)
... Removed 28 stack frames]]>
    </failure>
  </testcase> <!-- UserLoginAPI_Test -->
</testsuite> <!-- UseLoginTest -->
